templates/base/packages/nextjs/next.config.js.template.mjs

```diff
@@ -0,0 +1,29 @@
+import { stringify, withDefaults } from '../../../utils.js'
+
+const contents = ({ ignoreTsAndLintBuildErrors, extraConfig }) =>
+`// @ts-check
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
What if we do something like this?


‘’’ts
const nextConfig = stringify({
…extraConfig[0],
reactStrictMode: true,
…
})
‘’’

Reply:
it doesn't work because `webpack` value of the `nextConfig` object is a function and `inspect` returns `webpack: [Function: webpack] `.

Reply:
Ohh I see! Sorry missed it, yeah I tried finding other alternatives but didn’t find any good 

So yeah let’s go with spread operator only 🙌